src/coreclr/vm/exceptionhandling.h

```diff
@@ -570,11 +465,6 @@ class ExceptionTracker : public ExceptionTrackerBase
 
 public:
 
-    static UINT_PTR FinishSecondPass(Thread* pThread, UINT_PTR uResumePC, StackFrame sf,
-                                     T_CONTEXT* pContextRecord, ExceptionTracker *pTracker, bool* pfAborting = NULL);
-    UINT_PTR CallCatchHandler(T_CONTEXT* pContextRecord, bool* pfAborting = NULL);
-
-    static bool FindNonvolatileRegisterPointers(Thread* pThread, UINT_PTR uOriginalSP, REGDISPLAY* pRegDisplay, TADDR uResumeFrameFP);
     static void UpdateNonvolatileRegisters(T_CONTEXT* pContextRecord, REGDISPLAY *pRegDisplay, bool fAborting);
 
     PTR_Frame GetLimitFrame()
```

Comment:
This should not be needed

Reply:
This still seems to be called from `ResetThreadAbortState`.

Reply:
That one is not needed either.

Reply:
For the FEATURE_EH_FUNCLETS

Reply:
I'll submit a follow up PR once this is merged. Rough preview: https://github.com/dotnet/runtime/commit/7891236d6c50f2453c86d300e66572ca50184761